picard/oauth.py

```diff
@@ -45,10 +45,35 @@
 )
 
 
+OOB_URI = 'urn:ietf:wg:oauth:2.0:oob'
+
+
+class InvalidStateError(Exception):
```

Comment:
I would prefix this class name.

```suggestion
class OAuthInvalidStateError(Exception):
```

Reply:
I renamed it and rebased